WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -189,4 +190,29 @@ private double GetPrivacyPercentage(CoinsView coins)
 		var pcPrivate = totalDecimalAmount == 0M ? 1d : (double)(privateDecimalAmount / totalDecimalAmount);
 		return pcPrivate;
 	}
+
+	private bool CanStartAutoCoinJoin(Wallet wallet)
+	{
+		if (!wallet.KeyManager.AutoCoinJoin)
+		{
+			return false;
+		}
+
+		if (IsUserInSendWorkflow)
+		{
+			return false;
+		}
+
+		if (wallet.ElapsedTimeSinceStartup < AutoCoinJoinDelayAfterWalletLoaded)
```

Comment:
This should be `<=`:

```suggestion
		if (wallet.ElapsedTimeSinceStartup <= AutoCoinJoinDelayAfterWalletLoaded)
```